api/src/main/java/org/openmrs/module/fhir/api/diagnosticreport/handler/AbstractHandler.java

```diff
@@ -1,6 +1,28 @@
 package org.openmrs.module.fhir.api.diagnosticreport.handler;
 
+import javax.naming.InvalidNameException;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.module.fhir.api.util.FHIRDiagnosticReportUtil;
 
```

Comment:
@harsha89 Other Handlers (e.g. LAB, RAD) will extend this Handler. This will contains default implementation of DiagReportHandler interface. The Users can decide whether they are going to use these default implementation or override those.